feature-libs/cart/wish-list/root/components/index.ts

```diff
@@ -0,0 +1 @@
+export * from './add-to-wishlist/index';
```

Comment:
this one is not exported in root's `index.ts`

Reply:
Good catch.  I added the export.